src/reducers/Material.js

```diff
@@ -19,6 +19,7 @@ function materialsUpdateOne(state, action) {
     const index = action.index || state.index;  // not passing index when modifying currently displayed material
     const material = action.material.clone();   // clone material to assert props re-render
     material.isUpdated = true;                  // to be used inside components
+    material.isNonPeriodic = action.material.isNonPeriodic;  // set the value of isNonPeriodic
```

Comment:
This doesn't do anything: `material` is already a clone of `action.material` ...

Reply:
During testing that was not the case. `material != action.material`.

[Please see this link for explanation](`https://exabyte.atlassian.net/wiki/spaces/SCI/pages/2408480846/Troubleshooting+Bugs`)